src/openfermion/utils/_rdm_mapping_functions.py

```diff
@@ -32,7 +32,7 @@ def map_two_pdm_to_one_pdm(tpdm, particle_number):
     Returns:
         opdm (numpy.ndarray): The 1-RDM contracted from the tpdm.
     """
-    return numpy.einsum('prrq', tpdm) / (particle_number - 1)
+    return numpy.einsum('prrq', tpdm) / float(particle_number - 1)
```

Comment:
Don't cast in these situations. It's unnecessary because `/` always means floating point division when `division` has been imported from `__future__`.